book__a__table.css

```diff
@@ -1,4 +1,4 @@
-.book__a__table__block {
+.book__a__table-block {

```

Comment:
аналогично комментарию выше

```suggestion
.book__a-table-block {

```

Reply:
либо вот так

```css
.book-a__table-block { 
```

Reply:
исправил - залил исправленное